hca/staging_area_validator.py

```diff
@@ -47,7 +47,7 @@ def __init__(
         staging_area: str,
         ignore_dangling_inputs: bool,
         validate_json: bool,
-        total_retries,
+        # total_retries,
```

Comment:
I am fine removing this if it's not working with our downstream dependencies for whatever reason, but I suggest removing the commented out bits altogether to avoid confusion and ticketing up re-adding them.